File: app/src/main/java/com/railian/maksym/provectustesttask/models/Picture.java

package com.railian.maksym.provectustesttask.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Picture {

    @SerializedName("large")
    @Expose
    private String large;
    @SerializedName("medium")
    @Expose
    private String medium;
    @SerializedName("thumbnail")
    @Expose
    private String thumbnail;

    public Picture(String thumbnail,String medium,String large){
        this.thumbnail=thumbnail;
        this.medium=medium;
        this.large=large;
    }

    public String getLarge() {
        return large;
    }

    public String getMedium() {
        return medium;
    }

    public String getThumbnail() {
        return thumbnail;
    }

}

File: app/src/main/java/com/railian/maksym/provectustesttask/models/Picture.kt
package com.railian.maksym.provectustesttask.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Picture(@field:SerializedName("thumbnail")
              @field:Expose
              val thumbnail: String, @field:SerializedName("medium")
              @field:Expose
              val medium: String, @field:SerializedName("large")
              @field:Expose
              val large: String)
